docs/atomic-sign.1.md

```diff
@@ -80,6 +80,12 @@ Now any image from the john repository will use the sigstore-staging location of
 means that signatures should be written to that location but read should occur from
 the http URL provided.
 
+The user's keyring will be used during signing. When running as root user this may
+not be desired. Another keyring may be specified gusing environment variable GNUPGHOME
+or set in configuration file atomic.conf. For example:
+
+default_keyring: /home/USER/.gnupg
```

Comment:
In my experience the `~` only expands to the current user at the shell. If used here it expands to the root users dir, not user who called sudo.